frontend/packages/console-app/src/components/user-preferences/language/LanguageDropdown.scss

```diff
@@ -0,0 +1,3 @@
+.co-user-preference-language-checkbox {
+margin-bottom: var(--pf-global--spacer--sm) !important;
```

Comment:
Not indented.

Reply:
Ahh! I thought the linter handles it.
Fixed indentation.

Reply:
Unfortunately, we don't use prettier or a linter for CSS yet :/ We have a story to add it.